code/modules/mob/living/carbon/martian/update_icons.dm

```diff
@@ -16,6 +16,8 @@
 	else
 		icon_state = icon_state_standing
 
+	update_inv_head(0)
```

Comment:
Now when update_icon is called, it shows what is being worn in the head on the HUD, but the icon overlay on the character is lost